test/unit/test_formula.py

```diff
@@ -684,3 +684,47 @@ def test_generate_formula_override_version():
     record_formula(formula_path, formula_filename, formula)
 
     assert '9.8.7' in formula
+
+
+def test_generate_formula_formula_includes():
+    """Tests that we generate the formula content correctly when all parameters are passed
```

Comment:
suggestion: we don't need to test all params here since we have other tests that cover that. What we'd be interested in for this test is only if the include line makes it into the generated formula and it's position and structure passed `brew audit`.